insecure/fixtures.go

```diff
@@ -49,6 +50,60 @@ func MessageFixture(t *testing.T, codec network.Codec, protocol Protocol, conten
 		Protocol:  protocol,
 	}
 
+	m := &Message{
+		Egress: egressMsg,
+	}
+
+	// creates corresponding event of that message that
+	// is sent by attack network to orchestrator.
+	e := &Event{
+		CorruptedNodeId:   originId,
+		Channel:           channel,
+		FlowProtocolEvent: content,
+		Protocol:          protocol,
+		TargetNum:         targetNum,
+		TargetIds:         targetIds,
+	}
+
+	return m, e, unittest.IdentityFixture(unittest.WithNodeID(originId), unittest.WithAddress("localhost:0"))
+}
+
+// IngressMessageFixture creates and returns a randomly generated gRPC ingress message that is sent between a corruptible conduit and the attack network.
+// It also generates and returns the corresponding application-layer event of that message, which is sent between the attack network and the
+// orchestrator.
+func IngressMessageFixture(t *testing.T, codec network.Codec, protocol Protocol, content interface{}) (*Message, *Event, *flow.Identity) {
+	// fixture for content of message
+	originId := unittest.IdentifierFixture()
+
+	var targetIds flow.IdentifierList
+	targetNum := uint32(0)
+
+	if protocol == Protocol_UNICAST {
+		targetIds = unittest.IdentifierListFixture(1)
+	} else {
+		targetIds = unittest.IdentifierListFixture(10)
+	}
+
+	if protocol == Protocol_MULTICAST {
+		targetNum = uint32(3)
+	}
+
+	channel := channels.Channel("test-channel")
```

Comment:
Same as above better to use channels.TestNetworkChannel

Reply:
replaced all occurrences https://github.com/onflow/flow-go/pull/2811/commits/10c85a68a545fc9d8d1341d9bd7ee7364e7b0fd0